requirements.txt

```diff
@@ -1,3 +1,5 @@
 opencv-python==4.10.0.84
 viam-sdk==0.29.1
 yt-dlp==2024.8.6
+pyinstaller
+google-api-python-client
```

Comment:
Why do we need to add google-api dependency here and in pyinstaller script now?

Reply:
My initial reasoning for doing this was because I was following along [here](https://docs.viam.com/how-tos/create-module/#compile-or-package-your-module) under the pyInstaller tab...however I tried to run the executable with and without the google-api-python-client and the stream does not load when it is not included in requirements.txt. I did some research and cannot figure out the exact reason why but my best understanding is that we want to make sure it is installed in the python env and bundled in the executable.